### PR TITLE
RM-8918 Upgrade MS dependencies

### PR DESCRIPTION
**Pull request by Frazerus**
https://re-motion.atlassian.net/browse/RM-8918

### CODE REVIEW

File: Remotion/Core/Development.Analyzers/Core.Development.Analyzers.csproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <Description>Analyzer ensuring best practices when using the Remotion assemblies.</Description>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Remove="$(MsBuildThisFileFullPath)"/>
+    <ProjectReference Remove="$(MsBuildThisFileFullPath)" />
```
Comment (a-ctor) [inline Remotion/Core/Development.Analyzers/Core.Development.Analyzers.csproj:12]: revert please as there are no other changes in this file

File: Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.NetFramework/ObjectBinding.Web.Development.WebTesting.TestSite.NetFramework.csproj [hunk 2]
```diff
@@ -56,8 +56,8 @@
     <Reference Include="log4net, Version=2.0.15.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\log4net.2.0.15\lib\net45\log4net.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Bcl.AsyncInterfaces.6.0.0\lib\net461\Microsoft.Bcl.AsyncInterfaces.dll</HintPath>
+    <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Bcl.AsyncInterfaces.7.0.0\lib\net462\Microsoft.Bcl.AsyncInterfaces.dll</HintPath>
```
Comment (a-ctor) [inline Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.NetFramework/ObjectBinding.Web.Development.WebTesting.TestSite.NetFramework.csproj:60]: If I am not mistaken, `Microsoft.Bcl.AsyncInterfaces` is an implicit dependency of `System.Text.Json`. So either we keep it on 6.0.0 or we push the `System.Text.Json` version to `7.x.x`.

File: Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.NetFramework/Web.config [hunk 3]
```diff
@@ -19,16 +19,15 @@
   </appSettings>
 
   <system.web>
-    <globalization culture="de-AT" uiCulture="en-US"/>
+    <globalization culture="de-AT" uiCulture="en-US" />
```
Comment (a-ctor) [inline Remotion/ObjectBinding/Web.Development.WebTesting.TestSite.NetFramework/Web.config:22]: revert the unrelated formatting changes here and in line 30?

File: Remotion/Web/Development.WebTesting.TestSite.NetFramework/Web.Development.WebTesting.TestSite.NetFramework.csproj [hunk 4]
```diff
@@ -53,8 +53,8 @@
     <Reference Include="log4net, Version=2.0.15.0, Culture=neutral, PublicKeyToken=669e0ddf0bb1aa2a, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\log4net.2.0.15\lib\net45\log4net.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
-      <HintPath>..\..\..\packages\Microsoft.Bcl.AsyncInterfaces.6.0.0\lib\net461\Microsoft.Bcl.AsyncInterfaces.dll</HintPath>
+    <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
```
Comment (a-ctor) [inline Remotion/Web/Development.WebTesting.TestSite.NetFramework/Web.Development.WebTesting.TestSite.NetFramework.csproj:56]: Same as the comment above regarding implicit package

**Review by a-ctor — CHANGES_REQUESTED** [review]

**Review by MichaelKetting — CHANGES_REQUESTED** [review]
Please recheck various Visual Studio artifects of the upgrade.

**Review by MichaelKetting — APPROVED** [review]

**Review by a-ctor — APPROVED** [review]
